taskfiles/lint.yaml

```
@@ -11,10 +11,12 @@ vars:
 
 tasks:
   check:
+    desc: "Runs the full suite of linters to identify warnings and violations."
     cmds:
       - task: "yaml-check"
 
   fix:
+    desc: "Runs the full suite of linters and automatically fixes any fixable issues."
```

```suggestion
    desc: "Runs the full suite of linters and fixes some violations."
```
Just matching the wording in spider.